pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -742,3 +755,26 @@ func (os *OpenStack) volumeService(forceVersion string) (volumeService, error) {
 		return nil, errors.New(err_txt)
 	}
 }
+
+func checkMetadataSearchOrder(order []string) error {
+	if order == nil {
```

Comment:
We should default to what we do currently for backward compat purposes.

Reply:
I'm currently setting the default on line 287. This conditional is for an scenario that can't happen on the current code, but keeps the function consistent by not trusting it's input.

Reply:
To read easily, can you set the default calue in readConfig()?

Reply:
To read easily, can you set the default value in readConfig()?

Reply:
moved the default value to `readConfig()`